java/src/Ice/src/main/java/com/zeroc/Ice/LocalException.java

```diff
@@ -4,13 +4,26 @@
 
 package com.zeroc.Ice;
 
-/** Base class for all Ice run-time exceptions. */
-public abstract class LocalException extends Exception {
+/** Base class for Ice run-time exceptions. */
+public class LocalException extends Exception {
   public LocalException() {}
 
+  public LocalException(String message) {
+    super(message);
+  }
+
   public LocalException(Throwable cause) {
```

Comment:
Not sure we need this cause-only constructor.

Reply:
Also removed. Now all constructors require a `String message`.